tidy3d/log.py

```diff
@@ -27,6 +29,8 @@
 
 DEFAULT_LOG_STYLES = {
     "DEBUG": None,
+    "SUPPORT": None,
```

Comment:
is the user able to set these logging levels? Would there be a use case for that? I'm wondering if it makes sense to hide them somehow to avoid possible confusion, or if we should just document them along with the other log levels

Reply:
They can set them in their local session, but as per Lucas' comment, there wouldn't really be any frontend messages printed at support or user level anyway.

Reply:
I see. I wonder if there's some way to document these two new levels, maybe as a short note in the config.logging_level description?

Reply:
So, yeah, I'm not sure what's best. I just pushed a commit with some more comments about the two new levels, what do you think?

Reply:
Looks good